app/models/user.rb

```diff
@@ -15,6 +15,7 @@ def logged_in?
 
   def create_notification_subscriptions(params)
     params[:subscription_types].each do |type|
+      Sendinblue::AddEmailByType.call(email, type)
```

Comment:
Toto treba presunut do `NotificationSubscription#confirm` a nebudes to musiet mat na dvoch miestach.